xtra-libp2p/src/endpoint.rs

```diff
@@ -530,6 +548,16 @@ impl Endpoint {
                                                     None => format!("Failed to connect with multi-address: {remote_addr}"),
                                                 }
                                             })?;
+
+                                        if blocked_peers.contains(&peer_id) {
+                                            tracing::info!(
+                                                target: "blocked_peers",
+                                                peer_id = %peer_id, // Weird but required
```

Comment:
❓ When using a target the syntax for fields changes? `%peer_id` does not work?

Reply:
Yea, just `%peer_id` tries to use the `Value` impl, which doesn't exist, instead of the `Display` impl